Backend/BlockChangesService.cs

```diff
@@ -24,6 +24,13 @@ public async Task<bool> AddAndFindMatch(BlockChangeRequest entityToAdd)
             return await MakeExchangeAndDeleteRequests(entityToAdd);
         }
 
+        public async Task<List<BlockChangeRequest>> FindWaitingStudentRequests(Guid studentId)
+        {
+            return await _blockChangesCollection.Find(
+                x => x.StudentId == studentId &&
+                     x.Status == ExchangeStatus.WaitingForExchange).ToListAsync();
+        }
+
```

Comment:
it is simple logic, so probably we don't need any unit test or do we?

Reply:
@marektavac Finally, I did a test